lib/smack/Prelude.cpp

```diff
@@ -327,7 +327,8 @@ struct IntOpGen::IntArithOp : public IntOp {
     if (!SmackOptions::BitPrecise ||
         (!SmackOptions::BitPrecisePointers && alsoUsedByPtr))
       funcs.push_back(getIntFunc(size));
-    if (SmackOptions::BitPrecise)
+    if (SmackOptions::BitPrecise ||
+        SmackOptions::BitPrecisePointers && alsoUsedByPtr)
```

Comment:
Please add parenthesis so that we do not have to rely on operator precedence (see above).

Reply:
> Please add parenthesis so that we do not have to rely on operator precedence (see above).

Done.